An entire "marketplace" feature (product listings, orders, creator dashboard) has tests referencing routes that don't exist in `app/` at all

`src/__tests__/marketplace-routes.test.tsx` requires six modules that do not exist anywhere in this repository: `@/app/marketplace/page`, `@/app/marketplace/create/page`, `@/app/dashboard/marketplace/page`, `@/app/api/marketplace/products/route`, `@/app/api/marketplace/orders/route`, `@/app/api/marketplace/orders/[orderId]/route`. Running the test suite fails every one of these with `Cannot find module '@/app/...'`. Two sibling files — `src/__tests__/marketplace.test.tsx` and `src/__tests__/marketplace-integration.test.tsx` — also fail (4 and 3 tests respectively), suggesting this isn't one orphaned test file but an entire feature area (product marketplace: listing, creation, checkout/orders, a creator-facing dashboard) that either (a) was planned and scaffolded with tests first but the implementation was never finished or was reverted, or (b) existed and was deleted at some point without anyone removing or updating the tests that depended on it.

This is a substantial amount of dead test surface (three files, 13 combined failures observed in this audit) actively failing in every CI run, which both wastes attention on "known failing" noise and risks masking a *real* new regression introduced in the same area going unnoticed against a backdrop of already-red tests.

**Task**: Determine which of the two scenarios this is. If the marketplace feature is genuinely planned/in-progress, either finish the missing routes or mark these three test files as explicitly skipped (`.skip`/`todo`) with a tracking reference, rather than leaving them failing silently in the noise. If the feature was abandoned or reverted, delete the three test files (and anything else marketplace-shaped that references now-nonexistent code) rather than leaving broken tests as the only remaining trace of it.

--- a/src/__tests__/marketplace-integration.test.tsx
+++ b/src/__tests__/marketplace-integration.test.tsx
@@ -1,18 +1,26 @@
 import { describe, it, expect } from "vitest";
+import * as marketplaceTypes from "@/types/marketplace";
+import { useMarketplace } from "@/hooks/useMarketplace";
+import { useCreatorMarketplace } from "@/hooks/useCreatorMarketplace";
+import * as CreatorStoreCard from "@/components/marketplace/CreatorStoreCard";
+import * as MarketplaceFilters from "@/components/marketplace/MarketplaceFilters";
+import * as ProductListingForm from "@/components/marketplace/ProductListingForm";
+import * as CheckoutFlow from "@/components/marketplace/CheckoutFlow";
+import * as ShippingAddressForm from "@/components/marketplace/ShippingAddressForm";
+import * as PaymentMethod from "@/components/marketplace/PaymentMethod";
+import * as OrderSummary from "@/components/marketplace/OrderSummary";
+import * as OrderManagement from "@/components/marketplace/OrderManagement";
+import * as OrderDetailsModal from "@/components/marketplace/OrderDetailsModal";
+import * as DigitalDelivery from "@/components/marketplace/DigitalDelivery";
 
 describe("Marketplace Integration", () => {
   it("should have all required types exported", () => {
     // Test that types can be imported
-    const types = require("@/types/marketplace");
-    
-    expect(types).toBeDefined();
-    expect(typeof types).toBe("object");
+    expect(marketplaceTypes).toBeDefined();
+    expect(typeof marketplaceTypes).toBe("object");
   });
 
   it("should have marketplace hooks available", () => {
-    const { useMarketplace } = require("@/hooks/useMarketplace");
-    const { useCreatorMarketplace } = require("@/hooks/useCreatorMarketplace");
-    
     expect(useMarketplace).toBeDefined();
     expect(typeof useMarketplace).toBe("function");
     expect(useCreatorMarketplace).toBeDefined();
@@ -21,20 +29,19 @@ describe("Marketplace Integration", () => {
 
   it("should have all marketplace components available", () => {
     const components = [
-      "@/components/marketplace/CreatorStoreCard",
-      "@/components/marketplace/MarketplaceFilters",
-      "@/components/marketplace/ProductListingForm",
-      "@/components/marketplace/CheckoutFlow",
-      "@/components/marketplace/ShippingAddressForm",
-      "@/components/marketplace/PaymentMethod",
-      "@/components/marketplace/OrderSummary",
-      "@/components/marketplace/OrderManagement",
-      "@/components/marketplace/OrderDetailsModal",
-      "@/components/marketplace/DigitalDelivery",
+      CreatorStoreCard,
+      MarketplaceFilters,
+      ProductListingForm,
+      CheckoutFlow,
+      ShippingAddressForm,
+      PaymentMethod,
+      OrderSummary,
+      OrderManagement,
+      OrderDetailsModal,
+      DigitalDelivery,
     ];
 
-    components.forEach((componentPath) => {
-      const component = require(componentPath);
+    components.forEach((component) => {
       expect(component).toBeDefined();
     });
   });

--- a/src/__tests__/marketplace-routes.test.tsx
+++ b/src/__tests__/marketplace-routes.test.tsx
@@ -1,45 +1,45 @@
 import { describe, it, expect } from "vitest";
+import MarketplacePage from "@/app/marketplace/page";
+import CreatePage from "@/app/marketplace/create/page";
+import DashboardPage from "@/app/dashboard/marketplace/page";
+import * as productsRoute from "@/app/api/marketplace/products/route";
+import * as ordersRoute from "@/app/api/marketplace/orders/route";
+import * as orderDetailRoute from "@/app/api/marketplace/orders/[orderId]/route";
 
 describe("Marketplace Routes", () => {
   it("should have marketplace page component", () => {
-    const MarketplacePage = require("@/app/marketplace/page").default;
     expect(MarketplacePage).toBeDefined();
     expect(typeof MarketplacePage).toBe("function");
   });
 
   it("should have create product page component", () => {
-    const CreatePage = require("@/app/marketplace/create/page").default;
     expect(CreatePage).toBeDefined();
     expect(typeof CreatePage).toBe("function");
   });
 
   it("should have creator dashboard page component", () => {
-    const DashboardPage = require("@/app/dashboard/marketplace/page").default;
     expect(DashboardPage).toBeDefined();
     expect(typeof DashboardPage).toBe("function");
   });
 
   it("should have products API route", () => {
-    const { GET, POST } = require("@/app/api/marketplace/products/route");
-    expect(GET).toBeDefined();
-    expect(POST).toBeDefined();
-    expect(typeof GET).toBe("function");
-    expect(typeof POST).toBe("function");
+    expect(productsRoute.GET).toBeDefined();
+    expect(productsRoute.POST).toBeDefined();
+    expect(typeof productsRoute.GET).toBe("function");
+    expect(typeof productsRoute.POST).toBe("function");
   });
 
   it("should have orders API route", () => {
-    const { GET, POST } = require("@/app/api/marketplace/orders/route");
-    expect(GET).toBeDefined();
-    expect(POST).toBeDefined();
-    expect(typeof GET).toBe("function");
-    expect(typeof POST).toBe("function");
+    expect(ordersRoute.GET).toBeDefined();
+    expect(ordersRoute.POST).toBeDefined();
+    expect(typeof ordersRoute.GET).toBe("function");
+    expect(typeof ordersRoute.POST).toBe("function");
   });
 
   it("should have order detail API route", () => {
-    const { GET, PATCH } = require("@/app/api/marketplace/orders/[orderId]/route");
-    expect(GET).toBeDefined();
-    expect(PATCH).toBeDefined();
-    expect(typeof GET).toBe("function");
-    expect(typeof PATCH).toBe("function");
+    expect(orderDetailRoute.GET).toBeDefined();
+    expect(orderDetailRoute.PATCH).toBeDefined();
+    expect(typeof orderDetailRoute.GET).toBe("function");
+    expect(typeof orderDetailRoute.PATCH).toBe("function");
   });
 });

